### DISCUSSION TITLE
A misorientation variation between two points of an EBSD map

### DISCUSSION THREAD

**Discussion by m-amrei** [mtex-toolbox/mtex#1690] [Ask Anything] [answered]
Hello, 
great thanks for this amazing tool (MTEX).
![Screenshot 2022-12-14 084317](https://user-images.githubusercontent.com/59927140/207610903-16c6576a-d560-49c0-ae5e-fa5285c6c07a.png)

I would like to have a linear graph of misorientation between two custom points on an EBSD map (for X1, Y1, and Z1). I understand that is already presented as colors on an EBSD map. There would be two uses for this: 
- Being able to verify the proper boundary angle.
- Representing the variation of crystals within a single grain.
Regards and thanks in advance for this, 
Mohsen

**Comment by kilir** [comment, marked as answer]
Hi,
please have a look at the documentation: https://mtex-toolbox.github.io/EBSDProfile.html
Cheers,
Rüdiger

**Comment by m-amrei** [comment]
Hello,

I get this error:


*Unable to use a value of type orientation as an index.*

When running this:
plot(ebsd_line.y,...
  angle(ebsd_line(1).orientations,ebsd_line.orientations)/degree)

can you please help me verify it?
my ebsd_line seems okay.
ebsd_line = EBSD (show methods, plot)

 Phase  Orientations  Mineral  Color  Symmetry  Crystal reference frame
     2     83 (100%)   AlphaP  Brown      m-3m

 Properties: bands, bc, bs, error, mad, reliabilityindex, x, y, grainId
 Scan unit : um
[image: image.png]

**Comment by m-amrei** [comment]
Fixed, it is okay. thanks
the angle was a variable in my case.
Regards,
Mohsen

**Comment by AzdiarGazder** [comment]
Hi @m-amrei,

Please have a look at the line profile function:
https://github.com/AzdiarGazder/mtexTools/tree/main/lineProfile

Warm regards,
Azdi
